recursion/recursionFibonacci/Solution1Recursion.go

```diff
@@ -0,0 +1,28 @@
+package FibonacciElements
+
+var slc []int
+
+type S1R struct{}
```

Comment:
use the name which includes impl (short form for implementation) say like ImplFibo1

Reply:
Just a suggestion only

Reply:
Changed to ImplS1R to be more clear